src/staging/gatsby-node.js

```diff
@@ -231,10 +237,65 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
       }
     }
   }
-  `
+  `;
+  await createRedirects({ graphql, actions });
+  await createParks({ graphql, actions });
+  await createPageSlugs("static", staticQuery, { graphql, actions, reporter });
+};
+
+const parkQuery = `
+{
+  allStrapiProtectedArea(filter: {isDisplayed: {eq: true}}) {
+    nodes {
+      id
+      orcs
+      protectedAreaName
+      slug  
+      url
+      oldUrl
+    }
+    totalCount
+  }
+}
+`;
+const staticQueryPath = `
+{ 
+  allStrapiLegacyRedirect {
+    nodes {
+      toPath
+      fromPath
+    }
+  }
+}`;
+
+
+async function createRedirects({ graphql, actions, result }) {
+  const response = await strapiApiRequest(graphql, staticQueryPath);
+  const redirects = response.data.allStrapiLegacyRedirect.nodes;
+
+  const resultPark = await strapiApiRequest(graphql, parkQuery);
 
-  await createParks({ graphql, actions })
-  await createPageSlugs("static", staticQuery, { graphql, actions, reporter })
+  let parks = resultPark.data.allStrapiProtectedArea.nodes;
+
+  parks.forEach((park) => {
+    redirects.map((item) => {
+
+      if (park.slug !== item.toPath) {
+        return actions.createRedirect({
+          fromPath: item.fromPath,
+          toPath: item.toPath,
+          isPermanent: true,
```

Comment:
We really just need fromPath and toPath.  The other options are for server-side redirects.